update_avatars.py

```
@@ -0,0 +1,155 @@
+    """Nexon API로 캐릭터 아바타 이미지 바이트를 반환한다. 실패 시 None."""
+    headers = {"accept": "application/json", "x-nxopen-api-key": API_KEY}
+    try:
+        # 1단계: ocid 조회
+        resp = await client.get(
+            f"https://open.api.nexon.com/maplestory/v1/id?character_name={nickname}",
+            headers=headers,
+        )
+        resp.raise_for_status()
+        ocid = resp.json().get("ocid")
+        if not ocid:
+            raise ValueError("ocid not found")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)